src/pages/_app.js

```diff
@@ -1,29 +1,35 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
```

Comment:
'useEffect' is imported, but not used, as far as I can see.

Reply:
you are right